integreat_cms/cms/models/regions/region.py

```diff
@@ -694,16 +736,15 @@ def get_pages(
         To retrieve all pages independently of their archived-state, use the reverse foreign key
         :attr:`~integreat_cms.cms.models.regions.region.Region.pages`.
 
-        Per default, the returned queryset has some limitations because of the usage of
-        :meth:`~django.db.models.query.QuerySet.difference` and :meth:`~django.db.models.query.QuerySet.union`.
-        To perform the extra effort of returning an unrestricted queryset, set the parameter
-        ``return_unrestricted_queryset`` to ``True``.
-
         :param archived: Whether or not only archived pages should be returned (default: ``False``)
         :type archived: bool
 
         :param return_unrestricted_queryset: Whether or not the result should be returned as unrestricted queryset.
                                              (default: ``False``)
+
+                                             .. deprecated:: 2023.10.0
```

Comment:
Hmm, I don't think we need deprecation grace periods for our functions, just removing them without notice is fine I guess.
Also, please remove all usages of the `return_unrestricted_queryset` parameter since it's not needed anymore.